Sources/LogtoClient/LogtoClient/LogtoClient+AccessToken.swift

```diff
@@ -14,7 +14,12 @@ extension LogtoClient {
     }
 
     func getAccessToken(by refreshToken: String, for resource: String?) async throws -> String {
-        let key = buildAccessTokenKey(for: resource, scopes: [])
+        // Hard code profile scope when requesting token for userinfo
+        // Force pass in offline_access as scope to avoid ID Token being returned
+        let scopes = resource == nil ?
+            [LogtoUtilities.Scope.profile.rawValue] :
+            [LogtoUtilities.Scope.offlineAccess.rawValue]
+        let key = buildAccessTokenKey(for: resource, scopes: scopes)
```

Comment:
Since we don't support custom scopes in V1, maybe we can treat this `scopes` as `null`.